src/Middleware/HttpLogging/src/StatusCodeHelper.cs

```diff
@@ -0,0 +1,148 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using System.Net;
+
+namespace Microsoft.AspNetCore.HttpLogging;
+
+internal static class StatusCodeHelper
+{
+    public static string ToStatusString(int statusCode)
```

Comment:
I copied this from https://github.com/dotnet/aspnetcore/blob/main/src/Shared/Hpack/StatusCodes.cs as per a suggestion on #33251.